tests/subworkflows/icgc-argo-workflows/merg_sort_dup/test.yml

```diff
@@ -0,0 +1,13 @@
+- name: "merg_sort_dup"
+  command: nextflow run ./tests/subworkflows/icgc-argo-workflows/merg_sort_dup -entry test_merg_sort_dup --tools 'markdup' -profile docker -c ./tests/config/nextflow.config
```

Comment:
Addressed the comment of testing code style consistent with others

Reply:
Add test case with default:
```
nextflow run ./tests/subworkflows/icgc-argo-workflows/merg_sort_dup -entry test_merg_sort_dup -profile docker -c ./tests/config/nextflow.config
```